### PR TITLE
fixed tooltip

### PR DESCRIPTION
**Pull request by Scootworks**
issue: when choicesTooltips not defined, an empty tooltip appears

### CODE REVIEW

File: LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua [hunk 1]
```diff
@@ -134,9 +134,11 @@ function IconPickerMenu:Initialize(name)
 end
 
 function IconPickerMenu:OnMouseEnter(icon)
-    InitializeTooltip(InformationTooltip, icon, TOPLEFT, 0, 0, BOTTOMRIGHT)
-    SetTooltipText(InformationTooltip, LAM.util.GetStringFromValue(icon.tooltip))
-    InformationTooltipTopLevel:BringWindowToTop()
+    if icon.tooltip and icon.tooltip ~= "" then
```
Comment (sirinsidiator) [inline LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua:137]: Not sure if this check is enough? GetStringFromValue could return an empty string too. Maybe just store the result string in a local variable and check that instead?

Comment (Scootworks) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua:137]: GetStringFromValue could return an empty string, but even an empty string is not worth to show an empty tooltip

Comment (sirinsidiator) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua:137]: I understand that point, but you are not checking if GetStringFromValue returns an empty string right now, so you'd still show the tooltip in that case.

Comment (Scootworks) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua]: now i've changed it to check GetStringFromValue first

File: LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua [hunk 2]
```diff
@@ -134,9 +134,12 @@ function IconPickerMenu:Initialize(name)
 end
 
 function IconPickerMenu:OnMouseEnter(icon)
-    InitializeTooltip(InformationTooltip, icon, TOPLEFT, 0, 0, BOTTOMRIGHT)
-    SetTooltipText(InformationTooltip, LAM.util.GetStringFromValue(icon.tooltip))
-    InformationTooltipTopLevel:BringWindowToTop()
+    local tooltipText = icon.tooltip and LAM.util.GetStringFromValue(icon.tooltip)
+    if tooltipText and tooltipText ~= "" then
+        InitializeTooltip(InformationTooltip, icon, TOPLEFT, 0, 0, BOTTOMRIGHT)
+        SetTooltipText(InformationTooltip, text)
```
Comment (sirinsidiator) [inline LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua:140]: Shouldn't you pass `tooltipText`? `text` is not defined ;)
Other than that it looks good now.

Comment (Scootworks) [reply LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua]: fixed, thx for the hint

**Pull request merged**